flybywiresim/a32nx
# fix(adirs): ADIRUs not aligned starting on runway

Fixes #4506 (the issue description states many things and is a bit chaotic, but this seems to be the main issue).

## Summary of Changes
Fixes ADIRUs not always being aligned starting on the runway.

## Additional context



Discord username (if different from GitHub): SjotgunSjonnie#9623

## Testing instructions
- Ensure the ADIRUs are aligned on the runway and in other spawning locations where this is to be expected.
- Ensure the ADIRUs are not aligned at the gate and in other spawning locations where this is to be expected.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
